project/Publish.scala

```diff
@@ -30,7 +30,8 @@ object Publish extends AutoPlugin {
 
   override def projectSettings = Seq(
     bintrayOrganization := Some("akka"),
-    bintrayPackage := "com.typesafe.akka:akka-http_2.11"
+    bintrayPackage := "com.typesafe.akka:akka-http_2.11",
+    bintrayRepository := (if (isSnapshot.value) "snapshots" else "maven")
```

Comment:
This does risk doing the non-snapshot publish twice: when you create the latest commit on master and then tag it, travis will start 2 jobs, but both will see the tag and select 'maven' here. Perhaps just add a note to the release train to wait for this?

Reply:
Yes. And such situation became especially common once we started committing release notes to documentation. After that PR is merged the tag is created usually very quickly afterwards, possibly making both builds release builds.

We have a [note about that in the Alpakka release train checklist](https://github.com/akka/alpakka/commit/2e295782a02dbdd0ff84b9299ea63040b360d25c#diff-5aa61b36f0395eb9d1fd1d14debe2e5cR39). Will add it to akka-http as well.